src/components/Documentation/Markdown/lang/dvc.js

```diff
@@ -1,17 +1,16 @@
 /* global exports:readonly */
-
 'use strict'
```

Comment:
do we reuse this in blog? we need to move it outside then

Reply:
actually this is highlight js? it should not be used anymore at all

Reply:
Seems it is hightligth.js configs. Removed it.